db/sql/2_seattle.sql

```diff
@@ -12,4 +12,4 @@ CREATE TABLE IF NOT EXISTS seattle_officers (
 );
 
 COPY seattle_officers (badge,full_name,title,unit,unit_description,first_name,middle_name,last_name,date)
-FROM '/seed/SPD_Rosters_Historical.csv' DELIMITER ',' CSV HEADER;
+FROM '/seed/Seattle-WA-Police-Department_Historical.csv' DELIMITER ',' CSV HEADER;
```

Comment:
Good call on renaming these

Reply:
Something we may have to look into down the line is updating the api routes which just use the city names atm, as well as the "ID" returned by the DepartmentMetadata object as it just uses a three character designator and that'll get confusing fast if we start adding more policing orgs.

Reply:
Ah yea that's a really good point. Not urgent now but there's sure to be a lot of SPDs and TPDs out there

Reply:
I shall make a few issues for this.